src/main/java/buddybot/FileStorage.java

```diff
@@ -0,0 +1,93 @@
+package buddybot;
+
+import java.io.*;
```

Comment:
Considering replacing the wildcard import with the actual classes that you need to import to add clarity!

Reply:
Agreed! That will help to reduce the possibility of potential name conflicts, such as if the same class name is found in more than one package imported through wildcard